src/components/ArticleLanguage.js

```diff
@@ -33,14 +33,24 @@ export const ArticleLanguage = ({ lang, title, close, closeAll }) => {
     closeAll()
   }
 
+  const onKeyBackspace = () => {
+    const selectedElement = document.querySelector('[nav-selected=true]')
```

Comment:
extra bonus here, 
you can find the `current` in `useNavigation`, checkout how it is used in Search component.

Reply:
Oh gotta love hooks